sys-process/cpu-x/cpu-x-3.2.4.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils xdg
+
+MY_PN='CPU-X'
```

Comment:
I see you mix usage of `' '` and `" "` throughout the ebuild. Could you stick to one style please? I've never seen `' '` being used like this, and am not sure if it's not good for bash, so I'd advice sticking to default style of using `" "`.